path_processor.go

```diff
@@ -29,6 +29,11 @@ func (d defaultPathProcessor) ExtractParameters(r *Route, _ *WebService, urlPath
 		} else {
 			value = urlParts[i]
 		}
+		if hasCustomVerb(key) {
```

Comment:
is here need to check CustomVerbEnabled?

Reply:
yes, do you agree?

Reply:
yes, i should be checked